samples/apps/basic/basic.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+// CCF
+#include "ccf/app_interface.h"
+#include "ccf/common_auth_policies.h"
+#include "ccf/ds/hash.h"
+#include "ccf/http_query.h"
+#include "ccf/json_handler.h"
+#include "ccf/version.h"
+
+#include <charconv>
+#define FMT_HEADER_ONLY
+#include <fmt/format.h>
+
+using namespace std;
+using namespace nlohmann;
+
+namespace basicapp
+{
+  using RecordsMap = kv::Map<string, std::vector<uint8_t>>;
+  static constexpr auto PRIVATE_RECORDS = "records";
+
+  class BasicHandlers : public ccf::UserEndpointRegistry
+  {
+  public:
+    BasicHandlers(ccfapp::AbstractNodeContext& context) :
+      ccf::UserEndpointRegistry(context)
+    {
+      openapi_info.title = "CCF Basic App";
+      openapi_info.description =
+        "Lightweight application for benchmarking purposes";
+      openapi_info.document_version = "2.3.0";
```

Comment:
```suggestion
      openapi_info.document_version = "0.0.1";
```

Nitpick, but just so that we get a sense of versioning maybe?